includes/class-wc-admin-reports-sync.php

```diff
@@ -119,28 +119,36 @@ public static function clear_queued_actions() {
 		} else {
 			self::queue()->cancel_all( null, array(), self::QUEUE_GROUP );
 		}
+
+		return __( 'Analytics jobs have been cleared.', 'wc-admin' );
 	}
 
 	/**
-	 * Adds regenerate tool to WC system status tools API.
+	 * Adds regenerate and job clearing tools to WC system status tools API.
 	 *
 	 * @param array $tools List of tools.
 	 * @return array
 	 */
-	public static function add_regenerate_tool( $tools ) {
+	public static function add_debug_tools( $tools ) {
 		if ( isset( $_GET['page'] ) && 'wc-status' === $_GET['page'] ) { // phpcs:ignore WordPress.Security.NonceVerification
 			return $tools;
 		}
 
 		return array_merge(
 			$tools,
 			array(
-				'rebuild_stats' => array(
+				'rebuild_stats'        => array(
 					'name'     => __( 'Rebuild reports data', 'wc-admin' ),
 					'button'   => __( 'Rebuild reports', 'wc-admin' ),
 					'desc'     => __( 'This tool will rebuild all of the information used by the reports.', 'wc-admin' ),
 					'callback' => array( __CLASS__, 'regenerate_report_data' ),
 				),
+				'clear_analytics_jobs' => array(
+					'name'     => __( 'Analytics jobs', 'wc-admin' ),
```

Comment:
Open to name changes here- not sure which is the most user-friendly and easiest to understand.

`Pending jobs`
`Queued jobs`
`Background jobs`
`Pending actions`
...

Reply:
Pending analytics actions ?